cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -1129,4 +1129,11 @@
     public static final String EXTENSIONS_DIR = "operational.stats.extensions.dir";
     public static final String REFRESH_INTERVAL_SECS = "operational.stats.refresh.interval.secs";
   }
+
+  /**
+   * Constants for Replication
+   */
+  public static final class Replication {
+    public static final String CDAP_SHUTDOWN_TIME_FILENAME = "/tmp/cdap_shutdown_time";
```

Comment:
Would it be better, instead of `/tmp/` to use `System.getProperty("java.io.tempdir")`?